src/safeposix/dispatcher.rs

```diff
@@ -229,7 +229,7 @@ pub extern "C" fn lindrustfinalize() {
     let drainedcages: Vec<(u64, interface::RustRfc<Cage>)> = cagetable.drain().collect();
     drop(cagetable);
     for (_cageid, cage) in drainedcages {
-        cage.exit_syscall();
+        cage.exit_syscall(0);
```

Comment:
I think its worth adding EXIT_SUCCESS (and EXIT_FAILURE) to constants and replacing all these zeros.